processor.go

```diff
@@ -639,3 +641,14 @@ func valueOrDefault(a, b string) string {
 	}
 	return b
 }
+
+func (p *CertProcessor) deleteFailedCertIfNeeded(c Certificate, namespace string) {
+	cutoff := c.Metadata.CreationTimestamp.Time.UTC().AddDate(0, 0, 7)
+
+	if cutoff.Before(time.Now().UTC()) {
+		error := p.k8s.deleteCertificate(c, namespace)
```

Comment:
`error` is an internal interface in go.  It's generally bad form to use the same name for a variable, that's why there's an idiom of naming error variables `err`.

Reply:
Ah gotya, thanks. Will change that to err instead of error.